src/lib/cache/subCache.cpp

```diff
@@ -71,8 +71,8 @@ volatile SubCacheState subCacheState = ScsIdle;
 */
 EntityInfo::EntityInfo(const std::string& _entityId, const std::string& _entityType, const std::string& _isPattern)
 {
-  entityId     = _entityId;
-  entityType   = _entityType;
+  const std::string entityId     = _entityId;
+  const std::string entityType   = _entityType;
```

Comment:
Strange things happening with the comments, perhaps I added a comment in the commit page ...?

You will need to assign  entityId and entityType before the body:

    EntityInfo::EntityInfo(const std::string& _entityId, const std::string& _entityType, const std::string& _isPattern) :
         entityId(_entityId), entityType(_entityType)